TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/Person.cs

```diff
@@ -13,10 +20,11 @@ public class Person
     public DateTime? DeletedOn { get; set; }
     public required string? Trn { get; set; }
     public required string FirstName { get; set; }
-    public required string MiddleName { get; set; }
+    public required string? MiddleName { get; set; }
```

Comment:
We don't want this nullable - we've used empty string in places where there's no middle name.